Front-End/src/App.tsx

```diff
@@ -4,7 +4,7 @@ import "./App.css";
 import { ThemeToggleButton } from "./components/ThemeToggleButton";
 import { HomePage } from "./pages/home-page/HomePage";
 import QuestionPage from "./pages/questions-page/QuestionPage";
-
+import RecommendationPage from "./pages/recommendation/RecommendationPage";
```

Comment:
i would put a new line between import and the function just for the readability sake of it.

Reply:
Fixed.